src/SourceGenerators/Uno.UI.SourceGenerators/Uno.UI.SourceGenerators.csproj

```diff
@@ -91,10 +91,13 @@
 	<Target Name="_UnoToolkitOverrideNuget" AfterTargets="AfterBuild" DependsOnTargets="BuiltProjectOutputGroup" Condition="'$(UnoNugetOverrideVersion)'!=''">
 		<PropertyGroup>
 			<_baseNugetPath Condition="'$(USERPROFILE)'!=''">$(USERPROFILE)</_baseNugetPath>
-      <_baseNugetPath Condition="'$(HOME)'!=''">$(HOME)</_baseNugetPath>
+			<_baseNugetPath Condition="'$(HOME)'!=''">$(HOME)</_baseNugetPath>
+			
+			<_OverridePackageId>Uno.UI</_OverridePackageId>
+			<_OverridePackageId Condition="'$(UNO_UWP_BUILD)'=='false'">Uno.WinUI</_OverridePackageId>
 
-			<_TargetNugetFolder Condition="'$(TargetFramework)'=='net461'">$(_baseNugetPath)\.nuget\packages\Uno.UI\$(UnoNugetOverrideVersion)\tools\uno-sourcegen</_TargetNugetFolder>
-			<_TargetNugetFolder Condition="'$(TargetFramework)'=='netstandard2.0'">$(_baseNugetPath)\.nuget\packages\Uno.UI\$(UnoNugetOverrideVersion)\analyzers\dotnet\cs</_TargetNugetFolder>
```

Comment:
I guess we have the same issue for other projects (e.g, Lottie, Svg)

https://github.com/unoplatform/uno/blob/a6eda7c5d1ab8d2b34cfc443fbd54e9432b8cf16/src/AddIns/Uno.UI.Svg/Uno.UI.Svg.csproj#L53

https://github.com/unoplatform/uno/blob/a6eda7c5d1ab8d2b34cfc443fbd54e9432b8cf16/src/AddIns/Uno.UI.Lottie/Uno.UI.Lottie.csproj#L54

But that can be fixed in a separate PR for sure.

Reply:
Yes, we need to adjust most packages to use the same top-level props file.